specifyweb/specify/views.py

```diff
@@ -399,86 +400,156 @@ class ReplaceRecordPT(PermissionTarget):
     update = PermissionTargetAction()
     delete = PermissionTargetAction()
 
+@transaction.atomic
+def record_merge_fx(model_name: str, old_model_id: int, new_model_id: int) -> http.HttpResponse:
+    """Replaces all the foreign keys referencing the old record ID
+    with the new record ID, and deletes the old record record.
```

Comment:
There seems to be an extra `record` at the end of the function documentation (docstring)